docs/tech/docker-hub.md

```diff
@@ -4,7 +4,7 @@
 - Get an existing admin (normally SRE) to add you to the `praekeltfoundation` organisation
 - Create a new repository under the organisation for your new image
 - Add the `automation` team as a collaborator to your new repository with write access
-- Use the `travis encrypt` command locally to encrypt the `praekeltorgdeploy` Docker Hub password (from 1Password) - for example, encrypt `REGISTRY_PASS=supersecretpassword`
+- Use the `travis encrypt` command locally to encrypt the `praekeltorgdeploy` Docker Hub password (from 1Password) - for example, encrypt `REGISTRY_PASS=supersecretpassword`. NOTE: if you are using `travis-ci.ORG` this will suffice. If you are using `travis-ci.COM` you will need to use the `--pro` flag e.g. `travis encrypt --pro 'REGISTRY_PASS="secret"'`. At Praekelt.org we are switching to `travis.com`, but check which service is integrated with your particular github repository.
```

Comment:
s/will need/may need/? If it's working like it should then you shouldn't need it.

Also, Praekelt.org are not the people switching, Travis CI are. Maybe something like:
> Travis CI are switching open source projects over to `travis-ci.com` and new projects will use .com.